operator/src/cli/command.rs

```diff
@@ -107,6 +110,10 @@ pub mod fixtures {
             help = "Trust level as a fraction, e.g. '2/3'",
         )]
         pub trust_level: super::TrustThreshold,
+
+        /// Trusting period. [default: 2/3 of unbonding period]
+        #[clap(long)]
+        pub trusting_period: Option<u32>,
```

Comment:
Since `trust_level` + `trusting_period` are often bundled together (i.e. if one is present in the cli args, then so is the other), it is possible to move them both to a single struct, called something like `trust_options`. And then use flatten in the actual command struct so that the user can just input `--trust-level` and `--trust-period`. This reduces code duplication. Do you wanna do this refactor here or just merge this and then we can open a separate PR for the refactor?